src/styles/eleventy-patterns/index.scss

```diff
@@ -13,5 +13,24 @@
 
 body,
 html {
+  color: var(--EP_color-text);
   height: 100%;
 }
+
+body.no-scroll {
+  overflow: hidden;
+}
+
+body:not(.EP-nav-hidden) {
+  padding-top: var(--EP_nav-height); /* 1 */
+}
+
+a {
+  color: var(--EP_color-blue);
+  text-decoration: none;
```

Comment:
I'm not sure how I feel about removing the underline. All of this should be EP namespaced.